src/server/usage/classes/collector_set.js

```diff
@@ -90,10 +89,19 @@ export class CollectorSet {
           this._log.warn(`Unable to fetch data from ${collectorType} collector`);
         });
     });
+    return Promise.all(fetchPromises);
```

Comment:
Just wondering, is there a specific reason we keep using promises versus using async/await?

Reply:
Not really. This lets us not await each fetch one at a time. I could be wrong, but using `Promise.props` inside the loop has it run all the fetches at the same time.